### ISSUE TITLE
NPE in CoordinatesInputDialog

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#12616]
### Describe your problem!

From Google Play as beta feedback and flagged as new for 2022.01.22-RC:
```
java.lang.NullPointerException: 
  at cgeo.geocaching.ui.dialog.CoordinatesInputDialog.updateGUI (CoordinatesInputDialog.java:394)
  at cgeo.geocaching.ui.dialog.CoordinatesInputDialog.access$1400 (CoordinatesInputDialog.java:60)
  at cgeo.geocaching.ui.dialog.CoordinatesInputDialog$CoordinateFormatListener.onItemSelected (CoordinatesInputDialog.java:594)
  at android.widget.AdapterView.fireOnSelected (AdapterView.java:1397)
  at android.widget.AdapterView.dispatchOnItemSelected (AdapterView.java:1386)
  at android.widget.AdapterView.selectionChanged (AdapterView.java:1369)
  at android.widget.AdapterView.checkSelectionChanged (AdapterView.java:1558)
  at android.widget.AdapterView.handleDataChanged (AdapterView.java:1534)
  at android.widget.AbsSpinner.onMeasure (AbsSpinner.java:193)
  at android.widget.Spinner.onMeasure (Spinner.java:642)
  at androidx.appcompat.widget.AppCompatSpinner.onMeasure (AppCompatSpinner.java:436)
  at android.view.View.measure (View.java:27131)
  at android.view.ViewGroup.measureChildWithMargins (ViewGroup.java:7951)
  at android.widget.LinearLayout.measureChildBeforeLayout (LinearLayout.java:1552)
  at android.widget.LinearLayout.measureVertical (LinearLayout.java:842)
  at android.widget.LinearLayout.onMeasure (LinearLayout.java:721)
  at android.view.View.measure (View.java:27131)
  at android.view.ViewGroup.measureChildWithMargins (ViewGroup.java:7951)
  at android.widget.LinearLayout.measureChildBeforeLayout (LinearLayout.java:1552)
  at android.widget.LinearLayout.measureVertical (LinearLayout.java:842)
  at android.widget.LinearLayout.onMeasure (LinearLayout.java:721)
  at android.view.View.measure (View.java:27131)
  at android.widget.ScrollView.measureChildWithMargins (ScrollView.java:2362)
  at android.widget.FrameLayout.onMeasure (FrameLayout.java:194)
  at android.widget.ScrollView.onMeasure (ScrollView.java:701)
  at android.view.View.measure (View.java:27131)
  at android.view.ViewGroup.measureChildWithMargins (ViewGroup.java:7951)
  at android.widget.FrameLayout.onMeasure (FrameLayout.java:194)
  at android.view.View.measure (View.java:27131)
  at android.view.ViewGroup.measureChildWithMargins (ViewGroup.java:7951)
  at android.widget.FrameLayout.onMeasure (FrameLayout.java:194)
  at android.view.View.measure (View.java:27131)
  at android.view.ViewGroup.measureChildWithMargins (ViewGroup.java:7951)
  at android.widget.FrameLayout.onMeasure (FrameLayout.java:194)
  at com.android.internal.policy.DecorView.onMeasure (DecorView.java:1173)
  at android.view.View.measure (View.java:27131)
  at android.view.ViewRootImpl.performMeasure (ViewRootImpl.java:4187)
  at android.view.ViewRootImpl.measureHierarchy (ViewRootImpl.java:2909)
  at android.view.ViewRootImpl.performTraversals (ViewRootImpl.java:3204)
  at android.view.ViewRootImpl.doTraversal (ViewRootImpl.java:2618)
  at android.view.ViewRootImpl$TraversalRunnable.run (ViewRootImpl.java:9971)
  at android.view.Choreographer$CallbackRecord.run (Choreographer.java:1010)
  at android.view.Choreographer.doCallbacks (Choreographer.java:809)
  at android.view.Choreographer.doFrame (Choreographer.java:744)
  at android.view.Choreographer$FrameDisplayEventReceiver.run (Choreographer.java:995)
  at android.os.Handler.handleCallback (Handler.java:938)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loop (Looper.java:246)
  at android.app.ActivityThread.main (ActivityThread.java:8633)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:602)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1130)
```

### How to reproduce?

unclear

### Reproducible

Unclear

### c:geo Version

2022.01.22-RC

### System information

```text
Android 9, Android 11
5 occurrences, 4 users
```

**Comment by Lineflyer** [comment]
Another one which looks similar:

```
java.lang.NullPointerException: 
  at cgeo.geocaching.ui.dialog.CoordinatesInputDialog.updateGUI (CoordinatesInputDialog.java:394)
  at cgeo.geocaching.ui.dialog.CoordinatesInputDialog.access$1400 (CoordinatesInputDialog.java:60)
  at cgeo.geocaching.ui.dialog.CoordinatesInputDialog$CoordinateFormatListener.onItemSelected (CoordinatesInputDialog.java:594)
  at android.widget.AdapterView.fireOnSelected (AdapterView.java:1397)
  at android.widget.AdapterView.dispatchOnItemSelected (AdapterView.java:1386)
  at android.widget.AdapterView.selectionChanged (AdapterView.java:1369)
  at android.widget.AdapterView.checkSelectionChanged (AdapterView.java:1558)
  at android.widget.AdapterView.handleDataChanged (AdapterView.java:1534)
  at android.widget.AbsSpinner.onMeasure (AbsSpinner.java:193)
  at android.widget.Spinner.onMeasure (Spinner.java:642)
  at androidx.appcompat.widget.AppCompatSpinner.onMeasure (AppCompatSpinner.java:436)
  at android.view.View.measure (View.java:27131)
  at android.view.ViewGroup.measureChildWithMargins (ViewGroup.java:7951)
  at android.widget.LinearLayout.measureChildBeforeLayout (LinearLayout.java:1552)
  at android.widget.LinearLayout.measureVertical (LinearLayout.java:842)
  at android.widget.LinearLayout.onMeasure (LinearLayout.java:721)
  at android.view.View.measure (View.java:27131)
  at android.view.ViewGroup.measureChildWithMargins (ViewGroup.java:7951)
  at android.widget.LinearLayout.measureChildBeforeLayout (LinearLayout.java:1552)
  at android.widget.LinearLayout.measureVertical (LinearLayout.java:842)
  at android.widget.LinearLayout.onMeasure (LinearLayout.java:721)
  at android.view.View.measure (View.java:27131)
  at android.widget.ScrollView.measureChildWithMargins (ScrollView.java:2362)
  at android.widget.FrameLayout.onMeasure (FrameLayout.java:194)
  at android.widget.ScrollView.onMeasure (ScrollView.java:701)
  at android.view.View.measure (View.java:27131)
  at android.view.ViewGroup.measureChildWithMargins (ViewGroup.java:7951)
  at android.widget.FrameLayout.onMeasure (FrameLayout.java:194)
  at android.view.View.measure (View.java:27131)
  at android.view.ViewGroup.measureChildWithMargins (ViewGroup.java:7951)
  at android.widget.FrameLayout.onMeasure (FrameLayout.java:194)
  at android.view.View.measure (View.java:27131)
  at android.view.ViewGroup.measureChildWithMargins (ViewGroup.java:7951)
  at android.widget.FrameLayout.onMeasure (FrameLayout.java:194)
  at com.android.internal.policy.DecorView.onMeasure (DecorView.java:1173)
  at android.view.View.measure (View.java:27131)
  at android.view.ViewRootImpl.performMeasure (ViewRootImpl.java:4187)
  at android.view.ViewRootImpl.measureHierarchy (ViewRootImpl.java:2909)
  at android.view.ViewRootImpl.performTraversals (ViewRootImpl.java:3204)
  at android.view.ViewRootImpl.doTraversal (ViewRootImpl.java:2618)
  at android.view.ViewRootImpl$TraversalRunnable.run (ViewRootImpl.java:9971)
  at android.view.Choreographer$CallbackRecord.run (Choreographer.java:1010)
  at android.view.Choreographer.doCallbacks (Choreographer.java:809)
  at android.view.Choreographer.doFrame (Choreographer.java:744)
  at android.view.Choreographer$FrameDisplayEventReceiver.run (Choreographer.java:995)
  at android.os.Handler.handleCallback (Handler.java:938)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loop (Looper.java:246)
  at android.app.ActivityThread.main (ActivityThread.java:8633)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:602)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1130)
```

**Comment by eddiemuc** [comment]
Fixed with PR #12617

**Comment by eddiemuc** [comment]
Wouldn't have happened with Kotlin btw :-)
